src/services/graph/hooks/exchange.ts

```diff
@@ -106,31 +107,19 @@ export function useOnePrice(variables = undefined, swrConfig: SWRConfiguration =
 
 export function useYggPrice(variables = undefined, swrConfig: SWRConfiguration = undefined) {
   const { chainId } = useActiveWeb3React()
-  const { data } = useSWR(
-    null,
-    () => getYggPrice(),
```

Comment:
formatting of unused functions (there are a few cases of this below) should be a separate commit